src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2767,7 +2767,7 @@ internal bool CanSaveImage(object parameters)
             return true;
         }
 
-        public void ShowSaveImageDialogAndSaveResult(object parameter)
+        public void ShowSaveImageDialogAndSave(object parameter)
```

Comment:
Although this is breaking API but we did mention to user that we could break view layer APIs

Reply:
@QilongTang so the users are aware they will have to modify view layer API's when needed?

Reply:
Yeah @reddyashish We made posts before for such clarification, do you prefer us obsolete it and make new APIs, we can do that as well.